static/js/tsconfig.json

```diff
@@ -13,6 +13,11 @@
         "jsx": "preserve",
         "sourceMap": true,
         "removeComments": false,
+        // Include the definitions (not polyfill)
+        "lib": ["es2015", "dom"],
+        // Allow tsc to find types included with packages and not in @types/*
+        "moduleResolution": "node",
```

Comment:
Oh _that’s_ what this is for. We can `yarn remove @types/source-map` then.

Reply:
When did we install `@types/source-map`?

Reply:
#12838. With neither `@types/source-map` nor `--moduleResolution=node`, `tsc` wouldn’t run.

Reply:
Oh cool.